README.md

```diff
@@ -54,19 +54,28 @@ Then to run the tests use ```npm test```
 
 On Windows, use ```npm run testwindows```
 
-# Caveats
+## Caveats
 
-## Two sync loops per sync frequency
+### Two sync loops per sync frequency
 Two sync loops may be invoked per sync frequency if the server-side sync frequency
 differs from the client-side frequency.
 
 This is because the client and server sync frequencies are set independently.
 Setting a long frequency on a client does not change the sync frequency on the
 server.
 
-The `syncFrequency` value of the dataset on the server should be set to the 
+The `syncFrequency` value of the dataset on the server should be set to the
 `sync_frequency` value of the corresponding dataset on the client to avoid this.
 
 For example:
   * `sync_frequency` on the client-side dataset is also set to 120 seconds.
   * `syncFrequency` on the server-side dataset is set to 120 seconds.
+
+## Sync API logging
```

Comment:
@evanshortiss like @wojta said, we've got another issue in progress to make additional changes to logging, specifically to allow a logger to be passed in, rather than the current default. I do like the sound of the idea of using the debug module as a default, if we have a default. That might be done in a separate issue again after FH-3219, I'm not sure.